pyproject.toml

```diff
@@ -5,17 +5,14 @@ description = "School-agnostic lib for implementing Student Success Tool workflo
 readme = "README.md"
 requires-python = ">=3.10,<3.13"
 dependencies = [
-    "databricks-connect~=16.0",
+    "databricks-connect",
     "databricks-sdk~=0.38.0",
     "faker>=20.1",
     "ipywidgets~=8.1",
     "matplotlib~=3.7",
     "missingno~=0.5",
     "mlflow~=2.13",
-    "numpy~=1.26",
-    "pandas~=2.1",
```

Comment:
Why have you removed `numpy` and moved `pandas`? We use them all over the place, and need to specify+constrain their versions -- especially important since `numpy` v2 breaks things.